x/claim/spec/README.md

```diff
@@ -0,0 +1,42 @@
+# `claim`
+
+## Abstract
+
+This specifies the `claim` module of Osmosis project, provide commands for claimable amount query and claim airdrop.
+We apply real-time decay after `DurationUntilDecay` pass where monthly decay rate is `-10%` of inital airdrop amount.
```

Comment:
Is the `-10%` a parameter or hard coded?

Reply:
Also feels odd to me that we don't just linearly decay until 0%

Reply:
What kind of decay mechanism is better for this?